src/server/pkg/storage/chunk/chunk.proto

```diff
@@ -15,10 +15,16 @@ message DataRef {
   int64 size_bytes = 4;
 }
 
+enum CompressionAlgo {
+  NONE = 0;
+  GZIP_BEST_SPEED = 1;  
+}
+
 message Ref {
   bytes id = 1;
   int64 size_bytes = 2;
   bool edge = 3;
 
-  // TODO: compression, encryption
+  bytes dek = 4;
```

Comment:
I think this should be spelled out. I knew this was a key associated with encryption, but I did not understand what the `de` part was. I'm guessing it is supposed to be an abbreviation for derived key, which I think we should just spell out here.

Reply:
It stands for Data Encryption Key.  The acronym is pretty widely used.
https://cloud.google.com/kms/docs/envelope-encryption

Reply:
The key is derived in our case, but it could also be randomly generated.